internal/printer/printer.go

```diff
@@ -141,6 +147,10 @@ func (p *printer) printCSSImports(cssLen int) {
 		p.print(fmt.Sprintf("import \"%s?astro&type=style&index=%v&lang.css\";", p.opts.Filename, i))
 		i++
 	}
+	if p.needsTransitionCSS {
+		p.addNilSourceMapping()
+		p.print(`import "astro/components/viewtransitions.css";`)
```

Comment:
I didn't clearly comment this yesterday, I was also wondering if we should create a new option for this import path, similar to how we have the `internalURL` option.

Not blocking as this is experimental and we can change again if we want.

Reply:
Ah yeah, I can add that, good call.

Reply:
Awesome. Do we also want to prefix with "experimental"? (just in case)